geometry_msgs/Point left_front
geometry_msgs/Point right_front
geometry_msgs/Point left_rear
geometry_msgs/Point right_rear
